app/components/DataTable/components/GeneViewer/index.jsx

```diff
@@ -39,22 +37,13 @@ const GeneViewer = (props) => {
   const {
     onClose,
     isOpen,
+    reportIdent,
     gene,
   } = props;
 
-  const { report } = useContext(ReportContext);
```

Comment:
was this causing problems ...? 

if you're using `reportIdent` btw, prop-types is complaining

Reply:
Some of the `DataTable`s are rendered within angular which can't access the react context

Reply:
is /report/61ec99d8-557f-44e7-aa00-27e2e5442bb5/small-mutations working for you when you click on a gene? seems like if we use both reportContext or reportIdent it comes back as null

Reply:
I am seeing that issue as well

Reply:
![image](https://user-images.githubusercontent.com/4342641/108267011-aae52180-711f-11eb-9f8b-34aaf75c15db.png)

Reply:
Fixed with latest commit